### ISSUE TITLE
InvalidImageFormatError after uploading an image in builder

### ISSUE THREAD

**Issue by ghing** [denverfoundation/storybase#298]
This is on Webfaction.

After uploading an image, the image does not appear in the builder.  Reloading the page causes the InvalidImageFormatError.

The image seems to be correctly loaded to MEDIA_ROOT/YYYY/MM/DD/filename.png

Here is the stack trace:

```
Traceback:
File "/home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  111.                         response = callback(request, *callback_args, **callback_kwargs)
File "/home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/django/views/generic/base.py" in view
  47.             return self.dispatch(request, *args, **kwargs)
File "/home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapper
  28.             return bound_func(*args, **kwargs)
File "/home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
  23.                 return view_func(request, *args, **kwargs)
File "/home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/django/utils/decorators.py" in bound_func
  24.                 return func(self, *args2, **kwargs2)
File "/home/floodlight/webapps/floodlight_staging/floodlight/apps/storybase_story/views.py" in dispatch
  484.         return super(StoryBuilderView, self).dispatch(*args, **kwargs)
File "/home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/django/views/generic/base.py" in dispatch
  68.         return handler(request, *args, **kwargs)
File "/home/floodlight/webapps/floodlight_staging/floodlight/apps/storybase_story/views.py" in get
  477.         context = self.get_context_data(object=self.object, **kwargs)
File "/home/floodlight/webapps/floodlight_staging/floodlight/apps/storybase_story/views.py" in get_context_data
  447.             context['story_json'] = mark_safe(self.get_story_json())
File "/home/floodlight/webapps/floodlight_staging/floodlight/apps/storybase_story/views.py" in get_story_json
  219.         to_be_serialized = resource.full_dehydrate(bundle)
File "/home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/tastypie/resources.py" in full_dehydrate
  745.                 bundle.data[field_name] = method(bundle)
File "/home/floodlight/webapps/floodlight_staging/floodlight/apps/storybase_story/api.py" in dehydrate_featured_asset_url
  183.         return bundle.obj.featured_asset_thumbnail_url(include_host=False)
File "/home/floodlight/webapps/floodlight_staging/floodlight/apps/storybase_story/models.py" in featured_asset_thumbnail_url
  268.             return featured_asset.get_thumbnail_url(**thumbnail_options)
File "/home/floodlight/webapps/floodlight_staging/floodlight/apps/storybase_asset/models.py" in get_thumbnail_url
  484.         thumbnail = thumbnailer.get_thumbnail(thumbnail_options)
File "/home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/easy_thumbnails/files.py" in get_thumbnail
  396.         thumbnail = self.generate_thumbnail(thumbnail_options)
File "/home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/easy_thumbnails/files.py" in generate_thumbnail
  292.                 "The source file does not appear to be an image")

Exception Type: InvalidImageFormatError at /build/479954cc181844cbbec1f1f3e86f4d42/
Exception Value: The source file does not appear to be an image
```

**Comment by ghing** [comment]
I traced this down to the call to `image.load()` at line 35 of /home/floodlight/webapps/floodlight_staging/venv/lib/python2.7/site-packages/easy_thumbnails/source_image.py in the function `pil_image`.  THe call to `image.load()` is raising an exception with the message "decoder jpeg not available".  It looks like when I installed PIL with pip, it didn't compile in JPEG support.  

It looks like http://community.webfaction.com/questions/7340/how-to-install-pil-with-truetype-support has instructions for building PIL with this support.
